src/components/SidebarMenu.tsx

```diff
@@ -20,16 +17,24 @@ const SidebarMenu = () => {
     <div className="sidebar-menu">
       <div className="top-section">
         <Link to="/">
-          <img className="logo" src="/open-poen-logo-blue.svg" alt="Home Logo" />
+          <img
+            className="logo"
+            src="/open-poen-logo-blue.svg"
+            alt="Home Logo"
+          />
         </Link>
       </div>
       <div className="middle-section">
         <ul>
           {linkData.map((linkInfo) => (
             <li
               key={linkInfo.name}
-              className={location.pathname === `/${linkInfo.name.toLowerCase()}` ? "active" : ""}
-              data-tooltip={linkInfo.tooltip} // Data-tooltip attribute
+              className={
+                location.pathname === `/${linkInfo.name.toLowerCase()}`
+                  ? "active"
+                  : ""
+              }
```

Comment:
I advise you to look towards NavLink, which itself determines the state of activity
https://reactrouter.com/en/main/components/nav-link